content/source/docs/enterprise/upgrade/index.html.md

```diff
@@ -10,85 +10,74 @@ If you used the legacy version of Terraform Enterprise (TFE), you probably have
 
-From **User Settings** > **Tokens** obtain the API key and set the environment variable `ATLAS_TOKEN`.
+## Step 3: Migrate Environments to the New Organization
 
-```bash
-export ATLAS_TOKEN=<YOUR_API_TOKEN>
-```
+Follow these steps for each legacy environment you want to migrate to your new organization.
 
-### 3.b. Configure the Backend for the Legacy Environment
+### Step 3.a: Ensure the Legacy Environment is Ready
 
-In most cases the backend is already configured to use the legacy environment. If the CLI commands `push`, `plan`, `apply`, or `state` were ever used, this block is already configured as needed.
+Navigate to your legacy TFE environment and ensure your environment is in a stable state before migrating. In particular:
 
-If a remote backend is not already configured with the TFE legacy environment, add the `backend` configuration block.
+- Make sure that no runs are currently in progress, and that the most recent plan has been applied.
+- Make sure that the environment is either unlocked, or locked by your currently logged-in user account.
+- Check with your colleagues and make sure no one needs to make changes to this infrastructure while you are migrating.
 
-```terraform
-terraform {
-  backend "atlas" {
-    name = "<LEGACY_ORG>/<SOURCE_ENVIRONMENT>"
-  }
-}
-```
+If you start migrating when there is a run in progress or the workspace is locked by a different user, the migration will fail.
 
-### 3.c. Reinitialize Terraform
+### Step 3.b: Create a New Workspace Using the "Import" Tab
 
-Reinitialize Terraform to ensure your working directory is up to date, especially if you just added backend configuration.
+Navigate to your new TFE organization, make sure you're on the [main workspaces list](../workspaces/index.html), and click the "+ New Workspace" button.
 
-```bash
-terraform init
-```
+On the new workspace page, there are two tabs beneath the "Create a new Workspace" header. Click the one labeled "Import from legacy (Atlas) environment".
 
-### 3.d. Reconfigure the Backend for the New Workspace
+![screenshot: form fields on the 'import from legacy (Atlas) environment' page](./images/new-workspace-import.png)
 
-Update the `name` field of the `backend` block to reference the new workspace.
+This import tab has fewer settings than the [new workspace tab](../workspaces/creating.html), since it migrates most settings from the legacy environment. Fill the fields as follows:
 
-```terraform
-terraform {
-  backend "atlas" {
-    name = "<NEW_ORG>/<DESTINATION_WORKSPACE>"
-  }
-}
-```
+- In the "Legacy Environment" field, enter the environment to migrate, in the form `<LEGACY ORGANIZATION>/<ENVIRONMENT NAME>`.
+- In the "Workspace Name" field, enter the new name of the workspace, which should usually be the same as the old environment name.
+- Choose the same VCS provider and repository as the old environment, or choose "None" if you plan to [push configurations via the API](../workspaces/run-api.html).
```

Comment:
Ah. [In the UI, we're linking to a different API page](https://github.com/hashicorp/atlas/blob/b97fb3cb3e0fe9d96c834034c5b2b0ab210bc5b5/frontend/atlas/app/templates/components/workspace-form.hbs#L108-L114), and this link is much more useful. Will fix the UI code!